guides/guide-for-adding-windows-node.md

```diff
@@ -1,229 +1,402 @@
 
-...
+Install the Tigera Calico operator and custom resource definitions.
 
-kube-system kube-flannel-ds-54954 1/1 Running 0 1m
+```bash
+kubectl create -f https://projectcalico.docs.tigera.io/manifests/tigera-operator.yaml
+```
+
+Install Calico by creating the necessary custom resource. For more information on configuration options available in this manifest, see the installation reference.
 
+```bash
+wget https://projectcalico.docs.tigera.io/manifests/custom-resources.yaml
+# After you make sure the CIDR is correct in the file
+# you can use:
+# vim custom-resources.yaml
+kubectl create -f custom-resources.yaml
 ```
 
-4. Add Windows Flannel and kube-proxy DaemonSets
+>  **Note:** Before creating this manifest, read its contents and make sure its settings are correct for your environment. For example, you may need to change the default IP pool CIDR to match your pod network CIDR.
 
-  
+Remove the taints on the master so that you can schedule pods on it.
 
-Now you can add Windows-compatible versions of Flannel and kube-proxy. In order to ensure that you get a compatible version of kube-proxy, you'll need to substitute the tag of the image. The following example shows usage for Kubernetes v1.24.3, but you should adjust the version for your own deployment.
+```bash
+kubectl taint nodes --all node-role.kubernetes.io/control-plane-
+```
 
-  
+It should return the following.
+
+```
+node/<your-hostname> untainted
+```
+3. Add calico and kube-proxy daemonsets from sig-windows-tools/hostprocess/calico
+
+Now you can add Windows-compatible versions of calico and kube-proxy. In order to ensure that you get a compatible version of kube-proxy, you'll need to substitute the tag of the image. The following example shows usage for Kubernetes v1.24.3, but you should adjust the version for your own deployment.
 
 ```bash
+git clone https://github.com/kubernetes-sigs/sig-windows-tools
+# Make sure to modify the version of kube-proxy image to match your kubernetes version
+# before applying it
+kubectl apply -f sig-windows-tools/hostprocess/calico/kube-proxy/kube-proxy.yml
+kubectl apply -f sig-windows-tools/hostprocess/calico/calico.yml
+```
 
-curl -L https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/kube-proxy.yml | sed 's/VERSION/v1.24.3/g' | kubectl apply -f -
+>  **Note** To find your version of kubernetes run the following command:
+> `kubeadm version`
 
-kubectl apply -f https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/flannel-overlay.yml
+5. Apply kube-calico-rbac.yml from sig-windows-tools/kubeadm/flannel
+Next you will need to apply the configuration that allows calico to spawn pods and keep them running:
 
+```bash
+kubectl apply -f sig-windows-tools/hostprocess/calico/kube-calico-rbac.yml
 ```
 
->  **Note** If you are using another version of kubernetes on your windows node, change v1.24.3 with your own version .
 
+### Joining a Windows worker node
```

Comment:
As the non-CNI-related steps on Linux and Windows as well as the verification steps are always the same, may think about consolidating them in a single paragraph instead of duplicating it multiple times